src/content/changes/highlights/2023-03-23.md

```diff
@@ -11,9 +11,11 @@ owner:
 
 ### Observability
 
-* prometheus-agent: we are in the process of deploying a new component to workload cluster which enables us to monitor your cluster and provides best quality of service. Some cluster upgrade might require manual intervention from our side, especially if you are using prometheus-operator. Feel free to ask team-atlas for any questions or help needed, also more details like which releases are affected can be found in our doc [here](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
-* grafana explore: we opened access to Grafana Explore feature which allow you to execute PromQL (and LogQL later) queries against all our metrics. See https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/
-* vertical-pod-autoscaler: v3.x is out which might break with your existing config. Also v3.2.0 might help regarding resources usage as VPA now autoscales itself. Fee free to ask team-atlas for any questions or help needed. See [github.com/vertical-pod-autoscaler](https://github.com/giantswarm/vertical-pod-autoscaler-app/releases/tag/v3.0.0)
+* prometheus-agent: cluster releases cointains a new component which enables us to monitor your cluster and provides best quality of service. Some cluster upgrade might require manual intervention from our side, especially if you are using prometheus-operator. Feel free to ask team-atlas for any questions or help needed, also more details like why is this needed and which cluster releases are affected can be found in our doc [here](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
+
+* grafana explore: Grafana Explore feature is now enabled. This allow you to execute PromQL (and LogQL later) queries against metrics we collect from your clusters. See https://docs.giantswarm.io/getting-started/observability/grafana/access/ https://grafana.com/docs/grafana/latest/explore/
+
+* vertical-pod-autoscaler: v3.x is out which requires to update existing config. Also v3.2.0 improves resources usage as VPA now autoscales itself. Fee free to ask team-atlas for any questions or help needed. See [github.com/vertical-pod-autoscaler@v3.0.0](https://github.com/giantswarm/vertical-pod-autoscaler-app/releases/tag/v3.0.0)
```

Comment:
```suggestion
* Vertical-Pod-Autoscaler (VPA): New version `v.3.x` is our with new improvements. Highlight that now VPA auto-scales itself. This version requires updating the existing config. Read the [migration docs](https://github.com/giantswarm/vertical-pod-autoscaler-app#moving-from-2xx-to-3xx---breaking-change) and feel free to ask our team in case of any doubt (@support-atlas).
```